app/pulp/app/serializers/task.py

```diff
@@ -27,7 +27,6 @@ class TaskSerializer(ModelSerializer):
     state = serializers.CharField(
         help_text="The current state of the task. The possible values include:"
                   " 'waiting', 'skipped', 'running', 'completed', 'failed' and 'canceled'.",
-        read_only=True
     )
```

Comment:
I just realized that if state isn't read_only, we probably want it to be a ChoiceField: http://www.django-rest-framework.org/api-guide/fields/#choicefield

I apologize for thinking of this at this point; if you think it would be better to open another issue, a seperate PR, or something, I'd be fine with that.